### DISCUSSION TITLE
Forward or asap planning not working in 6.25 community edition

### DISCUSSION THREAD

**Discussion by mpraga95** [frePPLe/frepple#558] [Q&A] [answered]
When I give a past date for a sale order and launch the web service it stops at 70% and doesn't plan the sale order. Reverse planning works fine for the same item. Can you help me how to fix this issue?

**Comment by jdetaeye** [comment]
This smells like a loop in your supply path, or some other issue that brings the solver in an infinite loop.

Changing the parameter "plan.loglevel" to 2 or higher should generate a more verbose log that allows you to pinpoint the issue more accurately.

**Reply by mpraga95** [reply]
With the same item's supply path configured I am able to create reverse sale order(web service runs fine and it plans the quantity too). Kindly help me to plan asap sale order. I have attached the log file here with reference to your previous comment.
[frepple-20221003111536.log](https://github.com/frePPLe/frepple/files/9696250/frepple-20221003111536.log)

**Comment by jdetaeye** [comment, marked as answer]
Any the following things should allow you to bring the solver out of this inefficient search;
- set a size_multiple of 1 on your operations to force discrete quantities.    
- set the "minimum shipment" on your demand equal to the demand quantity to enforce full deliveries.
- set the parameter "allowsplits" to "false".

**Reply by mpraga95** [reply]
Thank you for helping me with the issue, it got resolved when I set the parameter "allowsplits" to "false". 

->My approved manufacturing order is not getting planned in constrained plan. Any possible mistakes to correct it?
